js/router.js

```diff
@@ -183,7 +183,7 @@ module.exports = Backbone.Router.extend({
   cleanup: function(){
     $('#loadingModal').addClass('hide'); //hide modal if it is still visible
     messageModal.hide();
-    $('#obContainer').removeClass('modalOpen innerModalOpen box-borderDashed noScrollBar overflowHidden');
+    $('#obContainer').removeClass('modalOpen innerModalOpen customizeUserPage noScrollBar overflowHidden');
```

Comment:
For now, we could probably live with this, but ideally (and hopefully going forward), the router shouldn't have to do such css cleanup. Whenever a view is adding classes to elements outside of itself, it should be cleaning up after itself by removing those classes in it's remove() method. Also, if it's a cached view (or a descendant of one), it should also be doing the cleanup in 'cache-detached' as well (and then likely would need to restore the classes in 'cache-reattached').